framework/src/modules/network/defaults/config.js

```diff
@@ -152,26 +143,7 @@ const defaultConfig = {
 	required: ['wsPort', 'seedPeers'],
 	default: {
 		wsPort: 5000,
```

Comment:
We can keep other defaults as well?

Reply:
Without typesafety it will always lead the same problem that these defaults are became obsolete or different from P2P defaults.

Reply:
I know there is no type-safety there but still, it is good to have a list of config options to look at. If someone wants to change it in the future, they will have to check available p2p-lib config options instead of changing the value straight away in this config. Even if someone gives a wrong field name, it will not have any effect on the working of the library. But yes, it is also important to be consistent with the library when we change the config the options. Let's deal with this later.